File: Lab1/ProblemA.py
import sys

data_var = sys.stdin.read().strip().split()

t_var = int(data_var[0])

i_var = 1

while i_var <= t_var:

    n_var = int(data_var[i_var])
    
    if n_var % 2 == 0:
        sys.stdout.write(str(n_var) + " is an Even number.\n")
    else:
        sys.stdout.write(str(n_var) + " is an Odd number.\n")
    
    i_var += 1

File: Lab1/Java/ProblemA.java
import java.util.Scanner;

public class ProblemA {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        if (sc.hasNextInt()) {
            int t = sc.nextInt();
            for (int i = 0; i < t; i++) {
                int n = sc.nextInt();
                if (n % 2 == 0) {
                    System.out.println(n + " is an Even number.");
                } else {
                    System.out.println(n + " is an Odd number.");
                }
            }
        }
    }
}
